velox/exec/SetAccumulator.h

```diff
@@ -86,6 +86,45 @@ struct SetAccumulator {
     }
   }
 
+  void addNullIndex(const char* buffer) {
+    VELOX_CHECK(!nullIndex.has_value());
+    vector_size_t serializedNullIndex;
+    memcpy(&serializedNullIndex, buffer, kVectorSizeT);
```

Comment:
There are InputByteStream and OutputByteStream classes in velox/common/base/IOUtils.h which can be used here to make the code easier to read.

Reply:
I was able to use InputByteStream for deserialization functions. But since we random access during serialization(write), I didn't use OutputByteStream.